app/views/buyers/applications/_listing.html.slim

```diff
@@ -2,7 +2,7 @@ div id="bulk-operations" style="display: none;"
   h3 Bulk operations
   p
     ' You have selected
-    span class="count"
+    span.count>
     | applications and you can make following operations with them:
```

Comment:
text will be outside of span tag

Reply:
sorry, it's supposed to be outside of span tag

Reply:
that's desired.

Reply:
yes, that's what i saw after my first comment ;-0